spec/02-integration/02-cmd/03-reload_spec.lua

```diff
@@ -636,6 +636,8 @@ describe("key-auth plugin invalidation on dbless reload #off", function()
     assert.same("my-key", json.data[1].key)
     admin_client:close()
 
+    ngx.sleep(1)
```

Comment:
Is this needed to fix the test? The gateway is reloaded afterward and it should effectively apply all the config changes.
Please add a line of comment here.

Reply:
thanks for the reminder, I intended to dig deeper into this one. Updated with a better fix.